.github/workflows/sycl-linux-precommit.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       build_artifact_suffix: "default"
       build_cache_suffix: "default"
       # Docker image has last nightly pre-installed and added to the PATH
-      build_image: "ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build"
+      build_image: "ghcr.io/intel/llvm/sycl_ubuntu2404_nightly:latest"
```

Comment:
Relates to my previous question, shouldn't we use `alldeps` here?

Reply:
Answered there.